### DISCUSSION TITLE
unused argument (access_token = access_token)

### DISCUSSION THREAD

**Discussion by CheeseburgerML** [MRCIEU/TwoSampleMR#526] [Q&A]
Hi all,
When I tried to extract instruments, I got this eror message, could you please help me figure out what happened?
I have tried to update the version of TwoSampleMR and ieugwasr, these do not work for addressing this issue.
 exposure <- extract_instruments(outcomes = "ieu-a-299")
Error in ieugwasr::tophits(outcomes, pval = p1, clump = clump, r2 = r2,  : 
  unused argument (access_token = access_token)
Thank you so much in advance!

**Comment by niuGitHub123** [comment]
This problem appears again....
